File: app/src/main/java/com/example/vishistvarugeese/memory/Variables/Entity/Note.java
package com.example.vishistvarugeese.memory.Variables.Entity;

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

import java.io.Serializable;

@Entity
public class Note implements Serializable {

    @PrimaryKey(autoGenerate = true)
    private int id;

    private String title;
    private String description;
    private String timestamp;


    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getTimestamp() {
        return timestamp;
    }

    public void setTimestamp(String timestamp) {
        this.timestamp = timestamp;
    }

}

File: app/src/main/java/com/example/vishistvarugeese/memory/Variables/Entity/Note.kt
package com.example.vishistvarugeese.memory.Variables.Entity

import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey
import java.io.Serializable

@Entity
class Note : Serializable {
    @PrimaryKey(autoGenerate = true)
    var id = 0
    var title: String? = null
    var description: String? = null
    var timestamp: String? = null

}